src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -125,17 +124,15 @@ public void joinRoomDuplicateUserFailure() {
         firstUser.setNickname("rocket");
         User secondUser = new User();
```

Comment:
It's passed in as a UserDto to the join room request, but that's actually good that you brought this up, as I think it can simply be changed to a UserDto instead of a User